resource/ckb-miner.toml

```diff
@@ -32,11 +30,22 @@ dsn = "" # {{
 # testnet => dsn = "https://48c6a88d92e246478e2d53b5917a887c@sentry.io/1422795"
 # }}
 
-[miner]
+[miner.client]
 rpc_url = "http://127.0.0.1:8114/" # {{
 # _ => rpc_url = "http://127.0.0.1:{rpc_port}/"
 # }}
 block_on_submit = true
 
 # block template polling interval in milliseconds
 poll_interval = 1000
```

Comment:
Maybe a config to turn off status output?

Reply:
> Maybe a config to turn off status output?

I'm considering adding more status to the console output, let me add this configuration parameter with more status in another PR.